src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -292,7 +294,7 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
                     }
                 }
                 this.correctColumns();
-                if (this.spatialProperties.controls['isWkt'].value) {
+                if (!this.spatialProperties.controls['isWkt'].value) {
```

Comment:
I guess if you created a function regarding my comment above, you can use it here as well.